nifi-registry-core/nifi-registry-web-api/src/test/resources/application-ITSecureFile.properties

```diff
@@ -28,9 +28,9 @@ nifi.registry.client.properties.file: src/test/resources/conf/secure-file/nifi-r
 
 # Embedded Server SSL Context Config
 server.ssl.client-auth: need
-server.ssl.key-store: ./target/test-classes/keys/localhost-ks.jks
-server.ssl.key-store-password: localhostKeystorePassword
-server.ssl.key-password: localhostKeystorePassword
+server.ssl.key-store: ./target/test-classes/keys/registry-ks.jks
```

Comment:
Is there a reason for the inconsistent formatting of these keys? Ideally, I'd prefer if `ssl` was changed to `tls`, but I understand legacy naming. However, the `.` and `-` seem odd.

Reply:
These are spring boot application properties names that are getting interpreted by that framework:
https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html

Reply:
Agree on the unfortunate legacy naming used by all these libs/frameworks though. it's unfortunate 😞